common/storage/sql/statements.h

```diff
@@ -51,7 +51,7 @@ retcode_t iota_transactions_select_hashes_approvers_statement(
  * Milestone statements
  */
 
-retcode_t iota_milestone_insert_statement(const milestone_t* milestone,
+retcode_t iota_milestone_insert_statement(const iota_milestone_t* milestone,
```

Comment:
a bit annoying, but i think it should be `iota_statement_milestone_insert`

Reply:
100% agreed